addons/l10n_vn/migrations/17.0.2.0.2/post-migration.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
```

Comment:
Do as you wish with this line but it isn't required anymore: utf-8 is assumed by Python